std/file.d

```diff
@@ -2887,6 +2887,7 @@ private void copyImpl(const(char)[] f, const(char)[] t, const(FSChar)* fromz, co
 
 unittest
 {
+    import std.algorithm, std.file; // issue 14817
```

Comment:
Compiler bugs in local imports - std.algo becomes a bunch of local aliases sooo it shadows current top-level symbols.